polaris-react/.storybook/GridOverlay/GridOverlay.tsx

```diff
@@ -1,14 +1,14 @@
-import React, {useEffect, useState} from 'react';
-import debounce from 'lodash/debounce';
-// eslint-disable-next-line import/no-deprecated
-import {EventListener} from '../../src';
+import React, {useState} from 'react';
+import {tokens} from '@shopify/polaris-tokens';
+
+import {useEventListener} from '../../src';
 import {classNames} from '../../src/utilities/css';
 
 import './GridOverlay.css';
 
-const COLUMNS_SMALL = 4;
+const COLUMNS_SMALL = 6;
 const COLUMNS_LARGE = 12;
-const BREAKPOINT = 768;
+const BREAKPOINT = tokens.breakpoints['breakpoints-lg'].value;
```

Comment:
[`tokens.breakpoints` is converted to `rem`s](https://github.com/Shopify/polaris/blob/d2d5972d82519b3eca8cfe8bb56e2e4dfc40138e/polaris-tokens/src/tokens.ts#L69) for setting properties such as `max-width`. I plan to get [this PR](https://github.com/Shopify/polaris/pull/6407) out of draft today which would allow us to simplify the implementation to something like:

`const {lgUp} = useBreakpoints(); lgUp ? 12 : 6`

For now, I suggest handling the conversion to `em`s manually:

```
import {toEm} from '@shopify/polaris-tokens'

const BREAKPOINT = toEm(tokens.breakpoints['breakpoints-lg'].value);
```

Reply:
Heads up, `useBreakpoints` is available in the `next-breakpoints` branch.